### PR TITLE
Add clear warning to EIPs which are not Living or Final

### PR DESCRIPTION
**Pull request by feld**
This PR fulfills a request to add a warning to EIPs which are not finalized.

Open to recommendations on changes to language, color scheme, etc.

image

Requested in: https://github.com/ethereum-cat-herders/PM/issues/214

### CODE REVIEW

**Comment by axic** [comment]
This is nice!

I would take the opportunity though to differentiate based on the different statuses here, in terms of stability:
- Draft -- The above message.
- Review -- Something like "This EIP is in an (early) review stage. It is subject to changes, but feedback is appreciated."
- Last Call -- Something like "This EIP is in the last call for review stage. The authors wish to finalize the EIP and appreciate feedback."
- Final -- No warning
- Living -- No warning
- Stagnant -- "This EIP had no activity for at least 6 months."
- Withdrawn -- "This EIP has been withdrawn."

**Comment by feld** [comment]
Hello,

I have pushed changes following your recommendation. Here are screenshots showing examples of the rendered output:

￼
EIP-173 Contract Ownership Standard o

EIP-712 Ethereum typed structured data hashing and signing «
￼
EIP-86 Abstraction of transaction origin and signature «

￼￼
EIP-1702 Generalized Account Versioning Scheme «

￼
This EIP has been withdrawn

**Comment by poojaranjan** [comment]
This is very helpful, thank you!

File: _layouts/eip.html [hunk 1]
```diff
@@ -2,6 +2,28 @@
 layout: default
 ---
 
+{% if page.status == "Draft" %}
+  <div class="draft">
+    ⚠️ This EIP is not recommended for general use or implementation as it is likely to change.
+  </div>
+{% elsif page.status == "Review" %}
+  <div class="review">
+    📖 This EIP is in the review stage. It is subject to changes, but feedback is appreciated.
```
Comment (axic) [inline _layouts/eip.html:11]:
```suggestion
    📖 This EIP is in the review stage. It is subject to changes and feedback is appreciated.
```

**Review by axic — COMMENTED** [review]
Looks good to me, the text can always be changed/improved. Perhaps some other editors want to take a look at this, cc @MicahZoltu @lightclient.

(Did not choose approve in case it may just be auto-merged.)

**Review by MicahZoltu — APPROVED** [review]
This proposal is an improvement over what is currently in the repository.  I personally would like a bit stronger negative wording to drive people away from *using* any EIP that isn't Final, but that can be changed/debated in a future PR.

**Pull request merged**